src/styles/index.scss

```diff
@@ -1,3 +1,24 @@
-body {
+@import './utils/variable';
+@import './utils/mixins';
+@import './blocks/reset';
+@import './blocks/card';
+@import './blocks/stars';
+
+* {
```

Comment:
Don't use * selector (it impacts performance)
Better use ``html`` or ``body`` tag.
![image](https://github.com/mate-academy/layout_product-cards/assets/54860641/190a292e-8fb5-4f62-a95f-da36b7271224)